formats/MrssFormat.php

```diff
@@ -29,6 +29,7 @@ class MrssFormat extends FormatAbstract {
 
 	protected const ATOM_NS = 'http://www.w3.org/2005/Atom';
 	protected const MRSS_NS = 'http://search.yahoo.com/mrss/';
+	protected const ITUNES_NS = 'http://www.itunes.com/dtds/podcast-1.0.dtd';
```

Comment:
If we are adding this namespace, are there any other tags that it requires?

Reply:
At least the files produced validate, so technically perhaps not. Apple maybe requires also other `itunes:` tags to be present (like `itunes:image`) for the feed to be acceptable, but I don't think we need care about that.

Reply:
It is nice to add the namespace to the root element so that it does not have to be repeated on each element that uses it.